src/coreclr/jit/compiler.h

```diff
@@ -8280,8 +8280,8 @@ class Compiler
             reg     = REG_R11;
             regMask = RBM_R11;
 #elif defined(TARGET_WASM)
-            reg     = REG_R0;
-            regMask = RBM_R0;
+            reg     = REG_STK;
+            regMask = RBM_STK;
```

Comment:
```suggestion
            reg     = REG_NA;
            regMask = REG_NA;
```
If that runs into asserts, `R0` will need to be physically brought back.

Reply:
Have brought it back, just didn't seem to be a great idea, at least not in this PR.